app/frontend/src/views/form/Create.vue

```diff
@@ -3,20 +3,20 @@
     <h1 class="my-6 text-center">Create New Form</h1>
     <v-stepper v-model="creatorStep" class="elevation-0">
       <v-stepper-header class="elevation-0 px-0">
-        <v-stepper-step :complete="creatorStep > 1" step="1" class="pl-1">
-          Set up Form
-        </v-stepper-step>
+        <v-stepper-step :complete="creatorStep > 1" step="1" class="pl-1">Set up Form</v-stepper-step>
         <v-divider />
-        <v-stepper-step :complete="creatorStep > 2" step="2" class="pr-1">
-          Design Form
-        </v-stepper-step>
+        <v-stepper-step :complete="creatorStep > 2" step="2" class="pr-1">Design Form</v-stepper-step>
       </v-stepper-header>
 
       <v-stepper-items>
         <v-stepper-content step="1" class="pa-1">
           <v-form ref="settingsForm" v-model="settingsFormValid">
             <h1>Form Settings</h1>
             <FormSettings />
+            <v-card outlined class="pa-4">
+              <FormDisclaimer />
+            </v-card>
+            <v-checkbox class="mb-6" :rules="disclaimerRules" required label="I agree to the disclaimer and statement of responsibility for Form Designers"></v-checkbox>
```

Comment:
This could be <v-checkbox ... /> instead (prefer self-closing elements if possible)